components/tikv_alloc/Cargo.toml

```diff
@@ -22,7 +22,6 @@ tempdir = "0.3"
 
 [dependencies.jemallocator]
 version = "0.1.9"
-optional = true
```

Comment:
This means jemalloc will be compiled even when choosing system-alloc?

Reply:
PTAL @ZhangHanDong

Reply:
@kennytm  this should be.I forgot to consider this, let me fix it.